website/docs/d/identity_oidc_client_creds.html.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: "vault"
+page_title: "Vault: vault_identity_oidc_client_creds data source"
+sidebar_current: "docs-vault-datasource-identity-oidc-client-creds"
+description: |-
+  Reads client credentials from an OIDC Client provisioned in Vault
+---
+
+# vault\_identity\_oidc\_client\_creds
+
+Reads client credentials from an OIDC Client provisioned in Vault.
+
+~> **Important** All data retrieved from Vault will be
+written in cleartext to state file generated by Terraform, will appear in
+the console output when Terraform runs, and may be included in plan files
+if secrets are interpolated into any resource attributes.
+Protect these artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+## Example Usage
+
+```hcl
+resource "vault_identity_oidc_key" "key" {
```

Comment:
A key with name "default" is built-in to Vault 1.10, so this will be updating (not creating) that key. I think we can omit this resource block entirely and remove the reference in `vault_identity_oidc_client` below?